Create journal-of-tropical-life-science.csl

jtls opened the pull request.

libora commented:
@jtls do not use CC4.0 use CC3.0 
correct tag is
`<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>`
see https://forums.zotero.org/discussion/57246/creative-commons-40/

libora commented:
@jtls there is still missing URL for CC!

rmzelle commented:
@jtls, thanks for submitting a style!

Could you please take a close look at https://github.com/citation-style-language/styles/wiki/Style-Requirements?

In particular:

* the style needs a "documentation" link to the author guidelines (http://www.jtrolis.ub.ac.id/index.php/jtrolis/about/submissions#authorGuidelines). See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link
* `<category field="generic-base"/>` should become `<category field="biology"/>`
* please change

```
    <author>
      <email>jtls@ub.ac.id</email>
      <name>JTROLIS</name>
    </author>
    <contributor>
      <name>JTROLIS Secretary</name>
      <email>jtls@ub.ac.id</email>
    </contributor>
```

to 

```
    <author>
      <name>JTROLIS Secretary</name>
      <email>jtls@ub.ac.id</email>
    </author>
```

* change the `<rights/>` element to exactly:

```
<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
```

(see https://github.com/citation-style-language/styles/wiki/Style-Requirements#7---license)

rmzelle commented:
`<style ... default-locale="de-DE">`

You changed the style locale to German. Is that correct?

Also please delete either the author or the contributor.

jtls commented:
Pada 2016-05-03 21:29, Rintze M. Zelle menulis:
> <style ... default-locale="de-DE">
> 
> You changed the style locale to German. Is that correct?
> 
> Also please delete either the author or the contributor.
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub [1]
> 
> 	*
> 
> --
> This email was virus checked by UB Mail Gateway
> 
> 
> Links:
> ------
> [1]
> https://github.com/citation-style-language/styles/pull/2040#issuecomment-216547013

I have changed it to IN and deleted the contributor.
Thank you so much for your help

adam3smith commented:
the `default-locale` should be the language in which references should be in the journal in question. Looking at the linked instructions, that would seem to be English (`"en-US"`) not Indonesian, or am I missing something?

jtls commented:
Pada 2016-05-04 09:52, Sebastian Karcher menulis:
> the default-locale should be the language in which references should
> be in the journal in question. Looking at the linked instructions,
> that would seem to be English ("en-US") not Indonesian, or am I
> missing something?
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub [1]
> 
> 	*
> 
> --
> This email was virus checked by UB Mail Gateway
> 
> 
> Links:
> ------
> [1]
> https://github.com/citation-style-language/styles/pull/2040#issuecomment-216730019

Thank you, I will change it

rmzelle commented on journal-of-tropical-life-science.csl:
```diff
@@ -0,0 +1,169 @@
+    <terms>
+      <term name="et-al">et al</term>
+    </terms>
+  </locale>
```
Can you delete this `<locale>...</locale>` section as well?

rmzelle commented:
Great, almost there.

It also looks like you based this style off of the "Springer Basic (numeric, brackets)" style. If so, could you add a "template" link, like so:

`<link href="http://www.zotero.org/styles/springer-basic-numeric-brackets" rel="template"/>`

jtls commented:
Then, What should we do?

libora commented:
@jtls If your style is based on "Springer Basic (numeric, brackets)" style add template link as Rintze wrote 22 day ago.

libora commented:
@jtls correct tag is `<link href="https://www.zotero.org/styles/springer-basic-brackets" rel="template"/>` (without "numeric" in name)

jtls commented:
those link is not longer available . Can I change it to http://www.zotero.org/styles/wetlands, because it looks similar.

libora commented:
Which link is not available? (I am a little confused)

jtls commented:
Pada 2016-05-26 15:54, Libor Ansorge menulis:
> Which link is not available? (I am a little confused)
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub [1]
> 
> 	*
> 
> --
> This email was virus checked by UB Mail Gateway
> 
> 
> Links:
> ------
> [1]
> https://github.com/citation-style-language/styles/pull/2040#issuecomment-221814782

this template link 
(http://www.zotero.org/styles/springer-basic-numeric-brackets) is not 
longer available when I checked in zotero, but i found new link 
(http://www.zotero.org/styles/springer-basic-brackets) for my template 
link. finally, It's work!
I am happy ^^

rmzelle commented:
> (http://www.zotero.org/styles/springer-basic-numeric-brackets) is not
longer available when I checked in zotero, but i found new link
(http://www.zotero.org/styles/springer-basic-brackets) for my template
link.

Yeah, sorry, my fault. I only looked at the first commit, which used the http://www.zotero.org/styles/springer-basic-numeric-brackets link, but I'm not sure we ever had a style with that ID in the repository (see e.g. https://github.com/citation-style-language/styles/commits/master/springer-basic-brackets.csl). Anyway, that's correct now.

@jtls, I'm confused why you put back the `default-locale="in-IN"` instead of "en-US" and the German locale section. Did you do that by accident?

jtls commented:
I copied a wrong file
that was accident, i will fix it
Sorry

rmzelle commented:
Does the journal ever use et-al abbreviation, by the way?

http://www.jtrolis.ub.ac.id/index.php/jtrolis/article/view/315/pdf has this reference:

> 19. Martiny JBH, Bohannan BJM, Brown JH, Colwell RK,
Fuhrman JA, Green JL, Horner-Devine MC, Kane M,
Krumins JA, Kuske CR, Morin PJ, Naeem S, Øvreås L,
Reysenbach AL, Smith VH, Staley JT (2006) Microbial
biogeography: putting microorganisms on the map. Nature
Reviews 4: 102-112.

With the current settings (`<bibliography et-al-min="5" et-al-use-first="3" ...>`), the CSL style will abbreviate this to "Martiny JBH, Bohannan BJM, Brown JH et al."

jtls commented:
Pada 2016-05-27 20:24, Rintze M. Zelle menulis:
> Does the journal ever use et-al abbreviation, by the way?
> 
> http://www.jtrolis.ub.ac.id/index.php/jtrolis/article/view/315/pdf [1]
> has this reference:
> 
>> * Martiny JBH, Bohannan BJM, Brown JH, Colwell RK, Fuhrman JA,
>> Green JL, Horner-Devine MC, Kane M, Krumins JA, Kuske CR, Morin PJ,
>> Naeem S, Øvreås L, Reysenbach AL, Smith VH, Staley JT (2006)
>> Microbial biogeography: putting microorganisms on the map. Nature
>> Reviews 4: 102-112.
> 
> With the current settings (<bibliography et-al-min="5"
> et-al-use-first="3" ...>), the CSL style will abbreviate this to
> "Martiny JBH, Bohannan BJM, Brown JH et al."
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> --
> This email was virus checked by UB Mail Gateway
> 
> 
> Links:
> ------
> [1] http://www.jtrolis.ub.ac.id/index.php/jtrolis/article/view/315/pdf
> [2]
> https://github.com/citation-style-language/styles/pull/2040#issuecomment-222145312
> [3]
> https://github.com/notifications/unsubscribe/AR2DPeQCGXw-DxczOClBUOEGdce4QRt6ks5qFvChgaJpZM4IRiNe

Those was old style (Martiny JBH, Bohannan BJM, Brown JH, Colwell RK, 
Fuhrman JA, Green JL, Horner-Devine MC, Kane M, Krumins JA, Kuske CR, 
Morin PJ, Naeem S, Øvreås L, Reysenbach AL, Smith VH, Staley JT (2006) 
Microbial biogeography: putting microorganisms on the map. Nature 
Reviews 4: 102-112.). We started new style in volume 6 (Martiny JBH, 
Bohannan BJM, Brown JH et al.)

rmzelle commented:
Thanks! Sorry this took a while to accept.

I just removed the locale section (https://github.com/citation-style-language/styles/commit/641ae41b0ff5efe5173106be6eed20481cd07099), since the section shouldn't have any effect, and "us" is not a valid locale code anyway (for US English it's "en-US").

```  <locale xml:lang="us">
    <terms>
      <term name="et-al">et al</term>
    </terms>
  </locale>
```

jtls commented:
Pada 2016-05-30 10:42, Rintze M. Zelle menulis:
> Thanks! Sorry this took a while to accept.
> 
> I just removed the locale section (641ae41 [1]), since the section
> shouldn't have any effect, and "us" is not a valid locale code anyway
> (for US English it's "en-US").
> 
> ```
> 
>  et al
> 
> --
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> --
> This email was virus checked by UB Mail Gateway
> 
> 
> Links:
> ------
> [1]
> https://github.com/citation-style-language/styles/commit/641ae41b0ff5efe5173106be6eed20481cd07099
> [2]
> https://github.com/citation-style-language/styles/pull/2040#issuecomment-222406024
> [3]
> https://github.com/notifications/unsubscribe/AR2DPV0yVzTNnsCLopSWjipwVkIUue1qks5qGlymgaJpZM4IRiNe

So, I have to change it into en-US, right?
Sorry for my many questions, because it is my first time to make 
citation style.

rmzelle commented:
> So, I have to change it into en-US, right?

No. I deleted the section, since it wasn't needed. I merged the pull request, so you should be all set.
